release/pkg/operations/upload.go

```diff
@@ -81,7 +81,7 @@ func UploadArtifacts(r *releasetypes.ReleaseConfig, eksArtifacts map[string][]re
 			}
 
 			// If the artifact is a helm chart, skip the skopeo copy as it's handled separately.
-			if strings.HasSuffix(artifact.Image.AssetName, "helm") {
+			if artifact.Image.AssetName != "" && strings.HasSuffix(artifact.Image.AssetName, "helm") {
```

Comment:
Can you move this into the if block on line 88?

Reply:
And you can remove the empty string check.

Reply:
Is AssetName nil, because I think len(Assetname) == 0 is true for both "" and nil

Reply:
updated!

Reply:
I can remove the empty string check, but lemme doublecheck that works still.